samples/bot-sso-adaptivecard/nodejs/bots/botSSOAdaptiveCard.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { CardFactory, ActivityHandler } = require('botbuilder');
+const AdaptiveCardResponse = require('../resources/adaptiveCardResponseJson.json');
+const Options = require('../resources/options.json');
+
+class BotSSOAdativeCard extends ActivityHandler {
+    constructor() {
+        super();
+
+        this.onMembersAdded(async (context, next) => {
+            //Send welcome message when app installed
+            await this.sendWelcomeMessage(context);
+
+            // By calling next() you ensure that the next BotHandler is run.
+            await next();
+        });
+
+        this.onMessage(async (context, next) => {
+            const text = context.activity.text;
+
```

Comment:
remove extra space

Reply:
as usual one line space is there..
<img width="386" alt="image" src="https://user-images.githubusercontent.com/109147706/213899937-277f25ad-829e-49c5-8a1b-0bf0ca066c03.png">